Backend/app/user/user_router.py

```diff
@@ -118,5 +121,18 @@ async def face(user_email: str, face: UploadFile):
     await upload_facial_profile(user_email, face_file)
 
 
+@router.post("/login_face_recognition")
+async def login_face_recognition(user_email: str, face: UploadFile):
+    bytes_image = await face.read()
+    result = await compare_facial_profile(user_email, bytes_image)
+
+    if not "error" in result | result == False:
```

Comment:
en python los comparadores booleanos son "and", "or", "not". Acá tendría que ser `if not ("error" in result or result == False).